README.md

```diff
@@ -1,64 +1,70 @@
+Poseidon is a solution for archaeogenetic genotype data organisation.
 
-Detailed documentation for Poseidon can be found at [https://poseidon-framework.github.io](https://poseidon-framework.github.io)
+This standard defines the core components of the Poseidon package. Further details on [genotype data](https://poseidon-framework.github.io/#/genotype_data), the [.janno file](https://poseidon-framework.github.io/#/janno_details) and the [.ssf file](https://poseidon-framework.github.io/#/ssf_details) are documented on the Poseidon website.
 
-A changelog is available [here](https://poseidon-framework.github.io/#/changelog).
+A changelog for this standard is available on the website [here](https://poseidon-framework.github.io/#/changelog).
 
-## The Poseidon package structure
+### The Poseidon package structure
 
-All ancient and modern data in Poseidon are distributed into so-called packages, which are directories containing a dedicated set of files. Packages correspond to published sets of genomes, or in case of unpublished projects, ongoing (and growing) sets of samples currently analysed. All text files in the package are UTF-8 encoded.
+A Poseidon package stores genotype data with context information for DNA samples from (ancient) (human) individuals. Packages are defined by the POSEIDON.yml file, which holds relative paths to all other files in a package.
 
-Every package should have the following files: 
+A package therefore must contain:
 
 - A `POSEIDON.yml` file to formally define the package
-- Genotype data in eigenstrat or plink format
-- A `.janno` file to store context information
+- Genotype data in PLINK or EIGENSTRAT format
+
+It should additionally contain:
+
+- A `.janno` file to store context information on spatiotemporal origin or sample quality
 - A `.bib` file for literature references
 
-It can also contain the following files:
+It can also contain:
 
-- A `README.md` file for arbitrary context information
+- A `README.md` file for arbitrary, additional context information
 - A `CHANGELOG.md` file to document changes to the package
 - A `.ssf` file with information on the underlying raw sequencing data
 
-Example:
+Here is an example of a package `Switzerland_LNBA_Roswita` in one directory:
 
 ```
 Switzerland_LNBA_Roswita/POSEIDON.yml
-Switzerland_LNBA_Roswita/Switzerland_LNBA.plink.bed
-Switzerland_LNBA_Roswita/Switzerland_LNBA.plink.bim
-Switzerland_LNBA_Roswita/Switzerland_LNBA.plink.fam
+Switzerland_LNBA_Roswita/Switzerland_LNBA.bed
+Switzerland_LNBA_Roswita/Switzerland_LNBA.bim
+Switzerland_LNBA_Roswita/Switzerland_LNBA.fam
 Switzerland_LNBA_Roswita/Switzerland_LNBA.janno
 Switzerland_LNBA_Roswita/Switzerland_LNBA.ssf
 Switzerland_LNBA_Roswita/Switzerland_LNBA.bib
 Switzerland_LNBA_Roswita/README.md
 Switzerland_LNBA_Roswita/CHANGELOG.md
 ```
 
-## The `POSEIDON.yml` file
+All text files in the package are UTF-8 encoded.
```

Comment:
Is this a requirement, or are you simply stating that is the case here? If the former, this should probably be phrased with a MUST/SHOULD kind of statement.

Reply:
Done. This is a MUST.